src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -56,7 +52,9 @@ export const Breadcrumbs = ({
       setShouldTruncate(willOverflow);
     };
 
-    const debouncedUpdateShouldTruncate = debounce(updateShouldTruncate, 200);
+    const debouncedUpdateShouldTruncate = debounce(updateShouldTruncate, 200, {
+      leading: true,
+    });
```

Comment:
The leading call

https://user-images.githubusercontent.com/86632227/224142032-204f2292-e55d-485b-91c5-7f3802d6bf8b.mov

Reply:
Actually gonna revert this since chrome now fails, and ff might be iframe edge case... not that many consumers of Breadcrumbs right now anyways